### DISCUSSION TITLE
WatchDog: rpi4-usb3-Critical RaspberryPi4+USB3+GPIO use dedected

### DISCUSSION THREAD

**Discussion by comtechnik** [OpenCCU/OpenCCU#2005] [🇩🇪 Fragen&Antworten]
### Describe the issue you are experiencing

Ich bekomme seit dem Update 3.65.11.20221005 immer diese Alarmmeldung:

Alarmname: WatchGog: rpi4-usb3
Status: ausgelöst

### Describe the behavior you expected

Beschreibung: Critical RaspberryPi4+USB3+GPIO use dedectet

### Steps to reproduce the issue

1.Kann sie bestätigen, ist dann weg, kommt aber immer wieder
2.
3.
...

### What is the version this bug report is based on?

3.65.11.20221005

### Which base platform are you running?

rpi4 (RaspberryPi4)

### Which HomeMatic/homematicIP radio module are you using?

RPI-RF-MOD

### Anything in the logs that might be useful for us?

```txt
nein
```

**Comment by Baxxy13** [comment]
Die relevante Information, ob wirklich was an einer USB3-Buchse hängt, fehlt leider. 
Ist dem so dann ist der Alarm gewollt, da die Nutzung der USB3-Buchsen für starke Funkstörungen bekannt ist.
Ist dem nicht so, dann ist es ein Fehlalarm. Unschön aber m.E. nicht 100%ig vermeidbar.

In beiden Fällen kannst du, nach eigenem Ermessen, den Alarm über die erweiterten Systemeinstellungen deaktivieren.
